workshops/forms.py

```diff
@@ -917,3 +917,39 @@ class Meta:
             'teaching_frequency_expectation': forms.RadioSelect(),
             'max_travelling_frequency': forms.RadioSelect(),
         }
+
+
+class AutoUpdateProfileForm(forms.ModelForm):
+    username = forms.CharField(disabled=True)
+
+    ERROR_MSG = ("This github account is already associated with different "
+                 "Amy account. Log out and log in using github account.")
+    github = forms.CharField(error_messages={'unique': ERROR_MSG})
+
+    class Meta:
+        model = Person
+        fields = [
+            'personal',
+            'middle',
+            'family',
+            'email',
+            'gender',
+            'may_contact',
+            'airport',
+            'github',
+            'twitter',
+            'url',
+            'username',
+            'affiliation',
+            'domains',
+            'lessons',
```

Comment:
This list seems to show up elsewhere (e.g., look for the comment "non-M2M-relationships" in views.py). If the lists actually are the same, can we stuff them into a constant somewhere so that everything stays in sync?